### PR TITLE
Mybranch

### PR DESCRIPTION
**Pull request by dhaliwal23**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,14 +12,15 @@ if(process.env.NODE_ENV!="production"){
   if(env.error) throw env.error;
 }
 
-const { Pool } = require('pg');
+const { Pool } = require("pg");
 var pool;
-const constring = process.env.DATABASE_URL || `postgres://${process.env.DB_USER}:${process.env.DB_PASS}@localhost/grababite`;
+const constring = process.env.DATABASE_URL || 'postgres://$process.env.DB_USER}:${process.env.DB_PASS)@localhost/grababite';
```
Comment (sanjayalwani) [inline index.js:17]: why was this changed?

File: index.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ app.get('/restaurant/:uid', (req, res) => {
 
 
 
-  .get('/RestaurantSearch',(request,response) =>{
+  app.get('/RestaurantSearch',(request,response) =>{
```
Comment (sanjayalwani) [inline index.js:71]: maybe a checkNotAuthenticated in here?

File: index.js [hunk 3]
```diff
@@ -121,4 +111,75 @@ app.post('/PostRestaurant', (request,response) =>{
 });
 
 
+
+
+
+
+
+
+app.get('/Searchpage',(request,response) =>{
```
Comment (sanjayalwani) [inline index.js:120]: checkNotAuthenticated here too perhaps?

File: views/pages/Search-RestUser.ejs [hunk 4]
```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+<head>
+  
+</head>
+
```
Comment (sanjayalwani) [inline views/pages/Search-RestUser.ejs:6]: There is no nav in this page, how are the users going to navigate to and from this page?

**Review by sanjayalwani — CHANGES_REQUESTED** [review]
How are the users going to reach the search page? I see no changes to the navigation 'nav.ejs' and it is not included in the search result page either

Can we also remove package-lock.json, I think it is in the gitignore so I wonder how it is getting in?

File: index.js [hunk 5]
```diff
@@ -121,4 +111,75 @@ app.post('/PostRestaurant', (request,response) =>{
...
+
+  app.post('/RestrSearch',(request,response) =>{
+    const {Svar}=request.body;
+   
+    pool.query('SELECT * FROM users',(error,results) =>{
+      if (error){
+        throw error;
+      }
+    
+    pool.query('SELECT * FROM restaurants WHERE name=$1 OR city=$1 OR address=$1',[Svar], (error,results2) =>{
```
Comment (mann-c) [inline index.js:166]: this query is not going to work because the user will have to add the exact address or name of the restaurant to find the restaurant, we need to do a LIKE comparison, also you need to look at the restaurant descriptions also

**Review by mann-c — CHANGES_REQUESTED** [review]
why is your branch 83 commits behind the master? you need to pull master onto your current branch, solve the merge conflicts that are occurring, this will require you to understand what the rest of the code in index.js is doing
